src/Hubbup.Web/launchSettings.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
Is this file needed in addition to https://github.com/Eilon/ProjectKIssueList/blob/master/src/Hubbup.Web/Properties/launchSettings.json ?